client/views/courses/edit/course.edit.html

```diff
@@ -18,8 +18,7 @@
 					</h2>
 					{{#unless isFrame}}
 						<span class="fa fa-info fa-fw" aria-hidden="true"></span>
-						{{mf 'course.propose.EventInfo' 'In case you want to publish a singular event, use the following form instead:'}}
-						<a href="/event/create">{{mf 'menue.CreateEvent' 'Create an event'}}</a>
+						{{mf 'course.propose.CourseInfo' 'An Openki course represents a short or long-term grassroots learning process on a more or less specific topic of interest. A new course could then range from a very draft idea inviting various collaborators to materialize it, to an already organized series of events (these are always attached to a specific course)'}}
```

Comment:
I don't really like the bunch of text which now appears in the proposal frame, to me it sounds too technical and is just too much. Maybe we could just add a question sign, which shows you a short explanation when hovering over it and leads you to the explanation of courses in the FAQ if clicked?

Reply:
yes, I agree there is too much text. But I would like that people read (a better version of) this text. Should I try to make it a little smaller? And then do as you see best. Just removing it completely is not a bad option either ...

Reply:
what means PR?

Reply:
PR = Pull-Request

Reply:
If I understood correctly, you will accept this PR without this specific change and I have do a new one? It would be better if you remove already the previous text since it is also fine without any explanation.

Reply:
@panosnethood you can still commit to the branch to add changes to this pull request. So if this is acceptable to you, just remove the text. (And create another branch with that change.)

Reply:
Yes sorry. I tried a smaller version first in case you prefer it, otherwise I would delete it for now.